src/linear_algebra/finite_dimensional.lean

```diff
@@ -850,6 +865,43 @@ by { unfold findim, simp [dim_top] }
 
 end top
 
+lemma eq_zero_of_findim_eq_zero [finite_dimensional K V]
+  (hV : findim K V = 0) (x : V) : x = 0 :=
+begin
+  haveI := findim_eq_zero_iff_subsingleton.mp hV,
+  exact subsingleton.elim x 0,
+end
```

Comment:
I think you should state and name this the same way as `dim_zero_iff_forall_zero`

Reply:
Done, it is now `findim_zero_iff_forall_zero`